Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.7-alpine as build
+WORKDIR /build
+RUN pip install virtualenv
+RUN virtualenv /build
+RUN apk add --update --no-cache \
+    python \
+    python-dev \
+    build-base \
+    linux-headers \
+    pcre-dev \
+    py-pip \
+    curl \
+    openssl \
+    git
```

Comment:
that's odd to have git inside a container

Reply:
We need git to install piperci-python